### PR TITLE
Add Live Search topic

### PR DESCRIPTION
**Pull request by keharper**
## Description

This PR migrates the Live Search events topic from devdocs to this repo.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by jeff-matthews — APPROVED** [review]
Looks good. Does this need an entry on the main index.md file?

**Comment by keharper** [comment]
Ah, good point. Product Recommendations also needs a blurb on the front page. I'll add them.

File: src/pages/index.md [hunk 1]
```diff
@@ -30,19 +31,27 @@ Use these sections to learn about how to develop PHP extensions.
...
+[Product Recommendations](product-recommendations/)
 
-Explore common tools and SDKs shared across Services for Adobe Commerce
+Learn how to use the Product Recommendations SDK with Adobe Commerce to fetch recommendations programmatically in the browser.
+
+<DiscoverBlock slots="link, text"/>
+
+[Commerce Reporting API](reporting/)
+
+Learn how to import data into your Commerce data warehouse and export raw data from your Commerce Reporting dashboards
```
Comment (jeff-matthews) [inline src/pages/index.md:54]:
```suggestion
Learn how to import data into your Commerce data warehouse and export raw data from your Commerce Reporting dashboards.
```

**Review by jeff-matthews — COMMENTED** [review]
One tiny suggestion about a period

**Pull request merged**